wallet/concepts/wallet-interoperability.md

```diff
@@ -10,6 +10,13 @@ A web dapp can integrate with multiple installed browser wallets simultaneously
 mechanism to the [`window.ethereum`](wallet-api.md#ethereum-provider-api) injected provider.
 This mechanism is enabled by using the standardized interfaces defined by EIP-6963.
 
+:::info Why EIP-6963?
+[EIP-1193](https://eips.ethereum.org/EIPS/eip-1193) standardized the wallet interface, but it suffers 
+from unpredictable conflicts when multiple wallets are installed due to how the provider object is injected. 
```

Comment:
I think this is in passive voice. Maybe this can be rewritten for active voice, such as:

EIP-1193 standardizes the interface for wallets, yet it encounters conflicts when multiple wallets are installed, stemming from the method of provider object injection. This can cause issues with wallet discovery, user onboarding, and connecting. The wallet discovery mechanism introduced by EIP-6963 solves these issues.

Reply:
Nice, I can make this change.